packages/client/components/ActivityLibrary/ActivityDetails.tsx

```diff
@@ -89,20 +86,6 @@ const query = graphql`
   }
 `
 
-interface DetailsBadgeProps {
```

Comment:
Moved this & its parents into a their own files. This component is getting a little wild